index.js

```diff
@@ -0,0 +1,565 @@
+
+const ngrok = "https://74b76b4a.ngrok.io/";
+const filepath = "/Users/ek/Projects/dasmitdersilbe/";
+//const filepath = "/home/student/e/eukaiser/Projects/dasmitdersilbe/";
+const appId = "amzn1.ask.skill.f3bc5473-cad0-42f2-8e82-47664796986d";
+
+// ffmpeg -i <input-file> -ac 2 -codec:a libmp3lame -b:a 48k -ar 16000 <output-file.mp3>
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const Alexa = require('alexa-sdk');
+const _ = require('lodash');
+const fs = require('fs');
+const hf = require('./helper_functions.js');
+const data = require('./data.js');
+const db = require('./database.js')
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.get('/', function (req, res) {
+  //res.send('hello');
+  fs.createReadStream(filepath + 'test.mp3').pipe(res);
+});
+
+app.get('/1', function (req, res) { fs.createReadStream(filepath + '1.mp3').pipe(res); });
```

Comment:
This seems like something a pattern would come in handy

Reply:
how can I refactor this?